### DISCUSSION TITLE
"Sched_name" not parsing the scheduler name correctly on history.txt

### DISCUSSION THREAD

**Discussion by M4CC** [azuritecoin/OnnxDiffusersUI#78] [General]
Hello,

"Sched_name" is not parsing the scheduler name correctly on history.txt.
Tried to use "scheduler" instead but it parses a bunch of info, actually only the "_class_name" is necessary here. 

I guess you can figure this out easily.
Thank you!

**Comment by azuritecoin** [comment]
Did you ever figure it out why the `sched_name` is not parsing correctly? I saw you had a pull request that you closed.
